### DISCUSSION TITLE
Custom column width

### DISCUSSION THREAD

**Discussion by ichuprov** [filamentphp/filament#8712] [General]
I'm having a problem setting a custom column width. In my column I am displaying 2 fields. One acts like a title and the 2nd item is like a description. For some reason the description pushes the columns out - that is, the description does not wrap at all. I have tried setting a max-width, which then does a weird thing of displaying the description text OVER the top of the remaining columns of the table to the right of it. Just trying to work out why this is happening. When I inspect the column, the width comes up with some ridiculous size. I don't understand why this one column is causing this issue. Any width sizing is being ignored when Filament is rendering the column.
The description field will contain HTML, that's why I'm using {!! !!} around it. It's very basic HTML, mainly paragraphs and simple formatting.

Here is my view snippet that I am using:

```
<div>
    {{$getRecord()->organisation_name}}<br>
    <div style="max-width: 250px !important; overflow-wrap: normal !important; font-size: 0.8em">
        <p>{!! $getRecord()->organisation_description !!}</p>
    </div>
</div>
```
Here is a screenshot of what is happening when it is displayed in the table view.
![image](https://github.com/filamentphp/filament/assets/1366482/feae53d2-012a-4d5a-96eb-758e783b4637)

**Comment by ichuprov** [comment]
Ok with the help of a friend I've worked it out. By default it has a style of white-space: none - that was the culprit. So I updated my snippet to include white-space: normal on my block of text, and that has now fixed the problem.

```
    <div style="white-space: normal !important; font-size: 0.8em">
        <p>{!! $getRecord()->organisation_description !!}</p>
    </div>
```

**Comment by howdu** [comment]
Column [wrapping content](https://filamentphp.com/docs/3.x/tables/columns/text#wrapping-content) can be enabled by:
```php
TextColumn::make('description')
    ->wrap()
```
